{
  "input": {
    "message": "LDAP bind failed: Invalid credentials (80090308: LdapErr: DSID-0C090449, comment: AcceptSecurityContext error, data 52e, v3839)",
    "sekoiaio": {
      "intake": {
        "dialect": "OpenVPN",
        "dialect_uuid": "e8ca856f-8a58-490b-bea4-247b12b3d74b"
      }
    }
  },
  "expected": {
    "message": "LDAP bind failed: Invalid credentials (80090308: LdapErr: DSID-0C090449, comment: AcceptSecurityContext error, data 52e, v3839)",
    "event": {
      "category": [
        "authentication"
      ],
      "kind": "event",
      "outcome": "failure",
      "type": [
        "start"
      ]
    },
    "sekoiaio": {
      "intake": {
        "parsing_warnings": [
          "No fields extracted from original event"
        ]
      }
    }
  }
}